test/integration/api.bats

```diff
@@ -13,24 +13,39 @@ function teardown() {
 }
 
 @test "docker build" {
-      start_docker 3
-      swarm_manage
-      
-      run docker_swarm images -q
-      [ "$status" -eq 0 ]
-      [ "${#lines[@]}" -eq 0 ]
+	start_docker 3
+	swarm_manage
+	
+	run docker_swarm images -q
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 0 ]
 
-      run docker_swarm build -t test $BATS_TEST_DIRNAME/testdata/build
-      [ "$status" -eq 0 ]
+	run docker_swarm build -t test $BATS_TEST_DIRNAME/testdata/build
+	[ "$status" -eq 0 ]
 
-      run docker_swarm images -q
-      [ "$status" -eq 0 ]
-      [ "${#lines[@]}" -eq 1 ]
+	run docker_swarm images -q
+	[ "$status" -eq 0 ]
+	[ "${#lines[@]}" -eq 1 ]
 }
 
-# FIXME
 @test "docker commit" {
-	skip
+	start_docker 3
+	swarm_manage
+
+	run docker_swarm run -d --name test_container busybox sleep 500
+	[ "$status" -eq 0 ]
+
+	# commit container
+	run docker_swarm commit test_container commit_image_busybox
+	[ "$status" -eq 0 ]
+	
+	# cluster refresh image during 30 seconds        
+	sleep 35
```

Comment:
Yeah, we should update the images after commit\pull\tag\load... I will check it and fix this problem.